sphinxcontrib/video/__init__.py

```diff
@@ -141,6 +143,15 @@ def run(self) -> List[video_node]:
             )
             preload = "auto"
 
+        controlslist: str = self.options.get("controlslist", "")
+        valid_controlslist_tokens = ["nodownload", "nofullscreen", "noremoteplayback", ""]
+        valid_controlslist = " ".join([token for token in controlslist.split() if token in valid_controlslist_tokens])
+        if controlslist != valid_controlslist:
+            logger.warning(
+                f'The provided controlslist ("{controlslist}") is not an accepted value. defaulting to "{valid_controlslist}"'
+            )
+            controlslist = valid_controlslist
+
```

Comment:
To make it less complicated I would not babysit the users and simply raise an error if a problem occur in the provided list. 

```suggestion
        controlslist: str = self.options.get("controlslist", "").split(",")
        valid_token = ["nodownload", "nofullscreen", "noremoteplayback"]
        if not set(control_list).issubset(valid_token):
            raise ValueError(f"The controlslist can only contains value from: {valid_token})
            
```